CONTRIBUTING.md

```diff
@@ -4,22 +4,38 @@
 
 If you would like to contribute enhancements or fixes, please do the following:
 
-1.  Fork the package repository.
-2.  Run `npm install` to setup all dependencies.
-3.  Hack on a separate topic branch created from the latest `master`.
-4.  Check for lint errors with `npm run lint` or use `linter-eslint` within Atom.
-5.  Run the package specs.
-6.  Commit the changes and push the topic branch to your fork.
-7.  Make a pull request to the main repo.
-8.  Welcome to the club!
+1. Fork the package repository and clone your fork to local file.
+
+1. Run `npm install` in the project to setup all dependencies.
+
+1. Run `apm link --dev` to install your working folder as a development-mode-only package in Atom.
+
+1. While on the `master` branch run `git checkout -b my-super-special-branch` to create and checkout a separate branch for your work. This makes merging your changes a lot easier for the maintainers.
+
+1. Hack away at the code, and don't forget to have fun.
+
+1. Check for lint errors with `npm run lint` or use `linter-eslint` within Atom.
+
+1. Run `atom path/to/cool-project --dev` to open Atom in developer mode and test-drive your improvements.
```

Comment:
`atom --dev path/to/cool-project` is the correct form (although Atom works with this form).

Reply:
Hmm. Yeah I guess you're right. `atom [options] [path...]`. Wow that really _is_ nitpicking. :wink:

Reply:
Atom didn't used to be that forgiving about which order that was 😉.